docs/api/basic.rst

```diff
@@ -0,0 +1,298 @@
+Basic Usage
+===========
+
+We are small package, so our API is not so big. There are only few classes, which are suggested for a basic usage.
+
+
+Request Classes
+---------------
+
+These are classes, that you use to send a request to server.
+
+
+.. autoclass:: mcstatus.server.MCServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. autoclass:: mcstatus.server.JavaServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+.. autoclass:: mcstatus.server.BedrockServer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Response Objects
+----------------
+
+But those were request classes, how about response ones?
+
+For Java Server
+***************
+
+.. class:: PingResponse
+    :canonical: mcstatus.pinger.PingResponse
+
+    The response object for :meth:`JavaServer.status() <mcstatus.server.JavaServer.status>`.
+
+    .. class:: Players
+        :canonical: mcstatus.pinger.PingResponse.Players
+
+        Class for storing information about players on the server.
+
+        .. class:: Player
+            :canonical: mcstatus.pinger.PingResponse.Players.Player
+
+            Just a class for a player information.
+
+            .. attribute:: name
+                :type: str
+                :canonical: mcstatus.pinger.PingResponse.Players.Player.name
+
+                Name of the player.
+
+            .. attribute:: id
+                :type: str
+                :canonical: mcstatus.pinger.PingResponse.Players.Player.id
+
+                ID of the player (in `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ format).
+
+        .. attribute:: online
+            :type: int
+            :canonical: mcstatus.pinger.PingResponse.Players.online
+
+            Current number of online players.
+
+        .. attribute:: max
+            :type: int
+            :canonical: mcstatus.pinger.PingResponse.Players.max
+
+            Maximum allowed number of players.
+
+        .. attribute:: sample
+            :type: list[Player] | None
+            :canonical: mcstatus.pinger.PingResponse.Players.sample
+
+            List of players, who are online. If server didn't provide this, it will be :obj:`None`.
+
+            .. note::
+                It's often empty or even contains some advertisement, because the specific server implementations or
+                plugins can disable providing this information or even change it to custom.
+
+                There is nothing what ``mcstatus`` can to do here.
+
+
+    .. class:: Version
+        :canonical: mcstatus.pinger.PingResponse.Version
+
+        A class for storing version information.
+
+        .. attribute:: name
+            :type: str
+            :canonical: mcstatus.pinger.PingResponse.Version.name
+
+            The version name, like ``1.19.3``. See
+            `MineCraft wiki <https://minecraft.fandom.com/wiki/Java_Edition_version_history#Full_release>`__
```

Comment:
```suggestion
            `Minecraft wiki <https://minecraft.fandom.com/wiki/Java_Edition_version_history#Full_release>`__
```

Minecraft doesn't have a capital `C`. There are multiple times this typo was made, please fix all of them.

Reply:
Fixed [here](https://github.com/py-mine/mcstatus/compare/afb0702d1c9fda079879cb0fb2a0b1ef92fb54f7..f33d6f5e6c80269ef9072380bd2966a4ca44843e). Thanks!